selenium_test/test/locators.py

```diff
@@ -10,6 +10,7 @@ class FirstPageLocators(object):
     BANNER = (By.XPATH, "//*[@class='page-header']/h1")
     APLUS_TEST_COURSE_INSTANCE_BUTTON = (By.XPATH, "//div[contains(@class, 'card')]/a[contains(@href, '/aplus1/basic_instance/')]//*[contains(@class, 'card-title')]")
     HOOK_EXAMPLE_BUTTON = (By.XPATH, "//div[contains(@class, 'card')]/a[contains(@href, '/aplus1/hook_instance/')]//*[contains(@class, 'card-title')]")
+    SHOW_LOGIN_BUTTON = (By.XPATH, "//*[contains(@class, 'show-extra-login-btn')]")
```

Comment:
The extra login button is not visible if Shibboleth and Google logins have not been enabled in the settings. The login page test might fail if the button does not exist, but that is easy to fix.